parser.go

```diff
@@ -0,0 +1,137 @@
+package voyager
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var noTxPrefix = regexp.MustCompile("^\\s*--\\s+(NO_TRANSACTION)")
```

Comment:
Back-ticks can help here, as it will be considered a raw string literal, so you don't need to escape twice.
 
```suggestion
var noTxPrefix = regexp.MustCompile(`^\s*--\s+(NO_TRANSACTION)`)
```